sql/wsrep_dummy.cc

```diff
@@ -36,7 +36,7 @@ const unsigned char* wsrep_xid_uuid(const XID*)
 }
 
 bool wsrep_prepare_key_for_innodb(THD* thd, const uchar*, size_t, const uchar*, size_t, struct wsrep_buf*, size_t*)
-{ return -1; }
+{ return 0; }
```

Comment:
Isnt -1 and 0 very different ? -1 will be converted to true , while 0 to false, Although it is never used but , still I will prefer 1 over 0

Reply:
Thanks!